controllers/hetznerbaremetalhost_controller.go

```diff
@@ -240,9 +243,21 @@ func (r *HetznerBareMetalHostReconciler) Reconcile(ctx context.Context, req ctrl
 	}
 
 	log = log.WithValues("HetznerBareMetalMachine", klog.KObj(hetznerBareMetalMachine))
-
 	ctx = ctrl.LoggerInto(ctx, log)
 
+	remediateConditionOfHbmm := conditions.Get(hetznerBareMetalMachine, infrav1.NoRemediateMachineAnnotationCondition)
+	if remediateConditionOfHbmm != nil && remediateConditionOfHbmm.Status == corev1.ConditionFalse {
```

Comment:
why do we not reconcile the host if the Machine is remediated? Then we also cannot deprovision anymore, can we?

Reply:
You said: 

> This condition is only mirrored from hbmm to hbmh, so I think this is fine.

We mirror the summary condition in the other direction, so that at least it would show on hbmm twice right?